src/Redux/api.tsx

```diff
@@ -466,6 +466,8 @@ const routes = {
   },
   getPatient: {
     path: "/api/v1/patient/{id}/",
+    TRes: Type<any>(),
```

Comment:
You do have the patient model defined in the codebase already right? You could use that instead of `any`

Reply:
Done